src/main/java/org/jboss/forge/arquillian/command/core/AddContainerCommand.java

```diff
@@ -125,6 +141,25 @@ public Result execute(UIExecutionContext context) throws Exception {
         return Results.success("Installed " + container.getName() + " dependencies");
     }
 
+    private void addTomcatConfiguration(Project project, ArquillianConfig config, String version, String profileId) {
```

Comment:
Other then that I don't have any other suggestions. I believe separation here would be good.